docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   postgres:
-    image: postgres
+    image: postgis/postgis
```

Comment:
I don't think this change is related to the task.

Reply:
it is not related, but i was needed to chage it to run my code, it was not changed when location is added